### PR TITLE
#163052923 Add Feedback features to Express API 

### PR DESCRIPTION
**Pull request by dayoaj**
#### What does this PR do?
Add features from feedback on Web API Development
#### Description of Task to be completed?
Rearrange files and folder
Add express-validator Middleware
Change functions to class (to follow es6 format)

#### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start`
* Using Postman test every endpoint above with this header:
 key: Content-Type value: application/json

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?

#163052923

### CODE REVIEW

**Comment by chukwuemekachm** [comment]
Please review your failing build.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,2 +1,4 @@
 {
+    "files.eol": "\n",
+    "prettier.endOfLine": "lf"
 }
```
Comment (chukwuemekachm) [inline .vscode/settings.json:4]: Add a new line to this file

File: server/models/db.json [hunk 2]
```diff
@@ -0,0 +1 @@
...eetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"5804dfc8-9543-48a3-961c-6fa02f7d108a","createdOn":"2019-01-07T05:03:22.477Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"25165214-4757-4522-8e20-7b434c0d4867","createdOn":"2019-01-07T05:04:40.341Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"}],"rsvps":[{"id":"cb9e78d3-853c-44b1-af7b-616411a8edcd","meetup":"6ecd62b7-e473-40e0-a6b1-eb9440b31b24","topic":"German Language Learners Group","response":"yes"},{"id":"b51b859a-ce4f-46b2-bc01-e5e98f00f81b","meetup":"77606213-7fd0-4d74-a8b6-c05c789b0f72","topic":"German Language Learners Group","response":"yes"},{"id":"8da0e5b7-3443-4db6-8329-9341a0f6bf7c","meetup":"77606213-7fd0-4d74-a8b6-c05c789b0f72","topic":"German Language Learners Group","response":"yes"}]}
```
Comment (chukwuemekachm) [inline server/models/db.json]: A very good implementation.
But you are to use in-memory storage.
Please replace the `db.json` file with a `db.js` file and define your data structure the same way.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Good work @dayoaj. Please attend to the comments below.
Thanks.

File: server/models/db.json (hunk 2 above)
Comment (dayoaj) [reply server/models/db.json]: Noted. Thanks

File: .vscode/settings.json (hunk 1 above)
Comment (dayoaj) [reply .vscode/settings.json:4]: Noted. Thanks for the feedback

**Review by chukwuemekachm — APPROVED** [review]
Nice.

**Pull request merged**